server/src/instant/intern/metrics.clj

```diff
@@ -201,6 +201,76 @@
      conn
      {:month-date (LocalDate/parse "2025-01-01")})))
 
+(defn format-date-label [date-val]
+  (if (instance? java.time.LocalDate date-val)
+    (.format date-val (java.time.format.DateTimeFormatter/ofPattern "MMM d"))
+    (let [sql-date (if (instance? java.sql.Timestamp date-val)
+                     (.toLocalDate (.toLocalDateTime date-val))
+                     (.toLocalDate date-val))]
+      (.format sql-date (java.time.format.DateTimeFormatter/ofPattern "MMM d")))))
+
+(defn rolling-avg-signups
+  "Get rolling 7-day average signup counts for the last n weeks"
+  ([conn weeks]
+   (let [end-date (LocalDate/now)
```

Comment:
If you use `LocalDate/now` in prod, I think that's going to give the date in UTC. Should we explicitly set the timezone to pacific?